.github/workflows/observer-e2e.yml

```diff
@@ -0,0 +1,108 @@
+name: Observer E2E
+
+on:
+  push:
+    branches:
+      - master
+      - 'auto-cadence-upgrade/**'
+      - staging
+      - trying
+      - 'feature/**'
+      - 'v[0-9]+.[0-9]+'
+  pull_request:
+    branches:
+      - master
+      - 'auto-cadence-upgrade/**'
+      - 'feature/**'
+      - 'v[0-9]+.[0-9]+'
+
+jobs:
+  golangci:
```

Comment:
is linting needed here?

Reply:
Indeed. Thank you for the review.
The original instructions we received targeted an Observer Microservice in a different repository. I was not sure whether this is still the goal, so I just removed the lint check from the flow-go repo.